### ISSUE TITLE
Autofix - remove trailing commas

### ISSUE THREAD

**Issue by weilbith** [webpro/knip#614]
Hey ✌🏾 

I know [the documentation](https://knip.dev/features/auto-fix/#whats-not-included) states that removing empty export declarations are not included in auto-fixes. But I wonder if maybe training commas could be removed. Here an example to better understand what I mean:

```typescript
export { isNotUsed, isActuallyUsed } from "./whatever"
```

If I run `knip --fix --fix-type exports` I'm left with this:

```typescript
export { , isActuallyUsed } from "./whatever"
```

In a more automated environment (i.e. git hooks), I could solve this by running auto-formatting on these files again. But maybe its quite feasible to do this right away. I seriously don't know how hard this is having the AST at hand. 🙈 

Thanks for any feedback to this idea. 🙏🏾

**Comment by weilbith** [comment]
Sorry, disregard that issue. I think my idea with the auto-formatting after `knip --fix` is the most feasible solution and does not bring up any additional conflicts with formatting configuration.
Because I stumbled across more scenarios and I think that's just not _knip_ its responsibility.

Example:
```typescript
export {
  isActuallyUsed,
  isNotUsed,
  alsoNotUsed,
} from "./whatever"
```
becomes
```typescript
export {
  isActuallyUsed,
  ,
  ,
} from "./whatever"
```
but should become after formattting
```typescript
export { isActuallyUsed } from "./whatever"
```

Sorry for the noise 🙈

**Comment by weilbith** [comment]
Though, is there a way to run `knip --fix` with some output, so I get the list of files that have actually been touched and I can just pipe them into the formatter script? 🤔

**Comment by webpro** [comment]
The only "structured" way to do this is `knip --fix --reporter json`, but the JSON reporter doesn't contain the `isFixed` boolean yet. If it's there you could pipe that and use e.g. `jq`, etc.

If this sounds useful to you and you're up for a PR, then this `issue` object has the `isFixed` property that can be added to the `json` output object: https://github.com/webpro/knip/blob/main/packages/knip/src/reporters/json.ts#L74

**Comment by weilbith** [comment]
Thank you very much. Very quick response again. 😊 
I realized that the whole topic is a little more complicated. Tools like `eslint` and `prettier` do not simply clean this up. And then again you might end up in a loop of running `knip --fix` then potentially remove code that has now become detected dead by `eslint` which might again trigger `knip` ♻️ 
Eventually (after I though implementing such a loop), I settled on this being not a big issue for continuous integration and trunk based development. More of a clean-up-old-code-base issue. So its probably not work the effort to implement all this stuff when it just happens any now and then and a human can solve it quickly. Git hooks will prevent a commit and a developer might need to take some laps to resolve the "recursive" effects.

Thanks for your kind help anyway! 🙏🏾
